browser/search_engines/tor_window_search_engine_provider_service.h

```diff
@@ -37,11 +37,9 @@ class TorWindowSearchEngineProviderService : public SearchEngineProviderService,
   void OnTemplateURLServiceChanged() override;
 
  private:
-  std::unique_ptr<TemplateURLData> GetInitialSearchEngineProvider(
-      PrefService* prefs) const;
   void ConfigureSearchEngineProvider();
 
-  std::unique_ptr<TemplateURL> alternative_search_engine_url_for_tor_;
+  std::unique_ptr<TemplateURL> default_templat_url_for_tor_;
```

Comment:
Nit: Typo, missing `e` in `template` :-)

Reply:
Fixed.